ui/hooks/useAccountTotalFiatBalance.js

```diff
@@ -54,10 +54,11 @@ export const useAccountTotalFiatBalance = (
   const primaryTokenImage = useSelector(getNativeCurrencyImage);
   const nativeCurrency = useSelector(getNativeCurrency);
 
-  const { loading, tokensWithBalances } = useTokenTracker({
+  const loading = false;
+  const { loading: tokensWithBalances } = useTokenTracker({
```

Comment:
This is some typo that got in somehow but fixed in my draft branch
```suggestion
  const { tokensWithBalances } = useTokenTracker({
```